alert-common/src/test/java/com/synopsys/integration/alert/common/persistence/model/job/details/DistributionJobDetailsModelJsonAdapterTest.java

```diff
@@ -14,10 +16,80 @@
 import com.google.gson.JsonDeserializationContext;
 import com.google.gson.JsonElement;
 import com.google.gson.JsonParseException;
+import com.google.gson.JsonSerializationContext;
 
-public class DistributionJobDetailsModelDeserializerTest {
+public class DistributionJobDetailsModelJsonAdapterTest {
     private final Gson gson = new Gson();
     private final JsonDeserializationContext jsonDeserializationContext = gson::fromJson;
+    private final JsonSerializationContext jsonSerializationContext = new JsonSerializationContext() {
+        @Override
+        public JsonElement serialize(Object o) {
+            return gson.toJsonTree(o);
+        }
+
+        @Override
+        public JsonElement serialize(Object o, Type type) {
+            return gson.toJsonTree(o, type);
+        }
+    };
+
+    // SERIALIZATION TESTS
```

Comment:
Perhaps another test class would be better than adding this section in comments??

Reply:
Should there be a test class per method being tested? I was just trying to add a quick visual guide for where certain tests for one method start and another method end.

Reply:
I thought the file was larger than it actually is. This should be fine.